stdlib/2and3/time.pyi

```diff
@@ -69,10 +69,11 @@ if sys.version_info >= (3, 3):
             ],
             _arg: Any = ...,
         ) -> struct_time: ...
-        @property
-        def tm_zone(self) -> str: ...
-        @property
-        def tm_gmtoff(self) -> int: ...
+        if sys.version_info >= (3, 6):
+            @property
+            def tm_zone(self) -> str: ...
```

Comment:
This does exist in 3.5 on Linux.

Reply:
The docs say that it's platform specific before Python 3.6 https://docs.python.org/3/library/time.html